### PR TITLE
Challenges

### PR DESCRIPTION
**Pull request by revirtux**

### CODE REVIEW

**Comment by revirtux** [comment]
update brach against master @AmitTPB

File: betasefer/urls.py [hunk 1]
```diff
@@ -26,3 +26,4 @@
 ] 
 urlpatterns += staticfiles_urlpatterns()
 urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
+urlpatterns += [path('silk/', include('silk.urls', namespace='silk'))]
```
Comment (Roey-coding) [inline betasefer/urls.py:29]: Please add a comment for "what is silk"

**Review by Roey-coding — CHANGES_REQUESTED** [review]
Wrote some reviews please fix

File: betasefer/urls.py (hunk 1 above)
Comment (AmitTPB) [reply betasefer/urls.py:29]: 👍

Comment (Roey-coding) [reply betasefer/urls.py]: por favor add new line at end of file

**Comment by Roey-coding** [comment]
@AmitTPB There is no documentation in the whole file, please add some of those

**Comment by AmitTPB** [comment]
> 
> 
> @AmitTPB There is no documentation in the whole files, please add some of those

did it

File: index/managers/usersmanger.py [hunk 2]
```diff
@@ -4,31 +4,127 @@
...
+            subjects[2],
+            player.houses.all(),
+            total,
+            player.image,
+            badges.get(player.nick, [])
+        ))
+
+    return sorted(users, key=lambda p: p.total, reverse=True)
+
+""" Deprecated by hexer
```
Comment (Roey-coding) [inline index/managers/usersmanger.py]: Please delete

File: index/views.py [hunk 3]
```diff
@@ -69,18 +69,21 @@ def dynamic_css(request, house_name):
...
                 'amount': len(challenges),
-                'category': category
+                'category': category,
+                'game_table': enumerate(game_table, 1),
+                'game_managers': game.managers.all() if game else []
             }, request))
     else:
-        return Http404("<p>No such category</p>")
\ No newline at end of file
+        return HttpResponse(Http404("<p>No such category</p>"))
```
Comment (Roey-coding) [inline index/views.py:89]: no new line

**Review by Roey-coding — CHANGES_REQUESTED** [review]
Delete all images

File: index/views.py [hunk 4]
```diff
@@ -69,18 +69,21 @@ def dynamic_css(request, house_name):
...
```
Comment (Roey-coding) [inline index/views.py:89]: This is an Error Lol, Please fix.

**Review by Roey-coding — CHANGES_REQUESTED** [review]

**Review by Roey-coding — APPROVED** [review]
🎉

**Pull request merged**